pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -1766,7 +1766,7 @@ var _ = Describe("Converter", func() {
 				Model: "ac97",
 			}
 			domain := vmiToDomain(vmi, c)
-			Expect(len(domain.Spec.Devices.SoundCards)).To(Equal(1))
+			Expect(domain.Spec.Devices.SoundCards).To(HaveLen(1))
 			Expect(domain.Spec.Devices.SoundCards).To(ContainElement(api.SoundCard{
```

Comment:
nit: you could use the `ConsistOf` matcher instead of asserting both length of the slice and containing member X.